lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -63,6 +63,12 @@ private_vlan:
   get_value: '/^feature private-vlan$/'
   set_value: "feature private-vlan"
 
+tacacs:
+  kind: boolean
+  get_command: "show running | i ^feature"
+  get_value: '/^feature tacacs\+$/'
+  set_value: "feature tacacs"
```

Comment:
Thanks for moving this into Feature. With luck we'll get the rest moved by 1.4.0.